docs/end-user-topics/ORCID.rst

```diff
@@ -0,0 +1,34 @@
+ORCIDs in Dockstore
+============================
+
+Have an ORCID_? You can now connect it to your Dockstore account.
+
+.. _ORCID: https://orcid.org/)
+
+Linking your ORCID to your Dockstore account
+--------------------------------------------
+
+The process is quite simple. If you are logged in now, you can `click here`_ to get there, or choose "Account" from the dropdown menu on the top right where your username is displayed. This will take you to a tab where you can control your linked services, such as Github and Quay. One of those, towards the bottom of the list, is the ability to link your account with ORCID.
+
+.. _click here: https://dockstore.org/accounts?tab=accounts
+
+.. image:: https://raw.githubusercontent.com/aofarrel/verbose-fiesta/master/Dockstore/Images/orcid_link_account_button.png
+  :alt: ORCID link account button
+
+From here, you'll be redirect to ORCID. With just a few clicks, you will now see this on the link services page of Dockstore.
+
+.. image:: https://raw.githubusercontent.com/aofarrel/verbose-fiesta/master/Dockstore/Images/orcid_after_account_linkage.png
+  :alt: After linking ORCID to Dockstore there will be options to relink, unlick, or view your token
+
+Now that your account is linked, when someone views an organization page of which you are a member, your ORCID will show up, helping connect your work on Dockstore with all of your other scientific contributions.
+
+.. image:: https://raw.githubusercontent.com/aofarrel/verbose-fiesta/master/Dockstore/Images/orcid_in_members_page.png
+  :alt: The author's ORCID showing up in the UCSC Genomics Institute members page on Dockstore
+
+You can also see the ORCID of members who have starred a workflow or tool. If you press the number next to the star button in the top right of a tool, you will see a list of stargazers. Stargazers with an ORCID will have that ORCID displayed.
+
+.. image:: https://raw.githubusercontent.com/aofarrel/verbose-fiesta/master/Dockstore/Images/orchid_stargazer.png
```

Comment:
Image seems bigger than normal. 
Regression for https://github.com/dockstore/dockstore/issues/3553 ?

Reply:
Do you want me to just scale down the image as a temporary fix?